spec/sidekiq/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -179,6 +182,20 @@ def expect_retryable_error(error_class)
               expect(submission.read_metadata(:ep_merge_pending_claim_id)).to be_nil
             end
           end
+
+          context 'when EP400 merge API call is disabled' do
```

Comment:
Tangential to the purpose of this PR: I realized that the test suite didn't cover the toggle's off state, so this new spec fixes that.